### ISSUE TITLE
Migrationsstore

### ISSUE THREAD

**Issue by ahausamm** [db-migrate/node-db-migrate#519]
I like your tool pretty much, but is it possible to use as store which migrations are done and which not a json file? Actually i don't like it that this settings are as a table in my database so i wondering if this is possible.

Thanks, Adrian

**Comment by wzrdtales** [comment]
Not yet and not really planned yet and this is very probable nothing you never ever want really.

The only reasonable place to store this data, is in the database being affected by those migrations. Why? B/c otherwise you have state that is distributed everywhere, but not where it is expected. This will drive you in bad scenarios where multiple workers start the same process at the same time, do not have the same state, or even break your applications. Also having this in a file would break the ability to catch concurrency alltogether.

**Comment by wzrdtales** [comment]
TL;DR; no go if your application wants to scale above anything greater than a single instance.

**Comment by ahausamm** [comment]
Oh, thanks for your quick answer.
